lib/jormungandr/src/Cardano/Wallet/Jormungandr/Api/Types.hs

```diff
@@ -107,9 +105,22 @@ data ApiStakeDistribution = ApiStakeDistribution
 instance ToHttpApiData BlockId where
     toUrlPiece (BlockId (Hash bytes)) = decodeUtf8 $ convertToBase Base16 bytes
 
-instance ToHttpApiData (ApiT ChimericAccount) where
-    toUrlPiece (ApiT (ChimericAccount bytes)) =
-        decodeUtf8 $ convertToBase Base16 bytes
+instance ToHttpApiData ApiAccountId where
+    toUrlPiece = toText
+
+instance FromText ApiAccountId where
+    fromText text = case Bech32.decodeLenient text of
+        Right (h, d) | h == accountIdHumanReadablePart ->
+            Right $ ApiAccountId d
+        _ ->
+            Left $ TextDecodingError "Invalid Jormungandr account ID."
+
+instance ToText ApiAccountId where
+    toText = Bech32.encodeLenient accountIdHumanReadablePart . getApiAccountId
+
+accountIdHumanReadablePart :: Bech32.HumanReadablePart
+accountIdHumanReadablePart =
+    Bech32.unsafeHumanReadablePartFromText "ca"
```

Comment:
I don't understand this choice of prefix :thinking: ? Is it simply provisional? Although it doesn't matter much for I believe Jörmungandr doesn't pay attention to what prefix is used, I'd suggest using something that is more representative of an account public key :+1:

Reply:
I just noticed that this is the default prefix used by `jcli` when generating addresses! 
This is unfortunate as this default is no longer in use, but it can be changed with a `--prefix` option !

Reply:
@KtorZ wrote:

> I don't understand this choice of prefix?  Is it simply provisional?

Good question. I had assumed that this was the expected prefix based on the contents of our `genesis.yaml`.

But you're right, Jörmungandr doesn't seem to care what the prefix is. Here's the output of a `jcli` session, when given one of the account addresses in our `genesis.yaml`:

```
$ jcli rest v0 account get ca1skzn99jx8a2rw80gp9uea47tmcndv7gm28vy9as6akevy32204aqwzpsctx --host http://127.0.0.1:46127/api --output-format json | jq -c
{"counter":0,"delegation":{"pools":[["5cfbb7f7ec0050499fbd1c79515216c5ad636b97d1944bde8a7db98e7d126803",1]]},"value":1}
```

Here's the output when given the same account address re-encoded with a different prefix (note that the 6-digit checksum has changed).
```
$ jcli rest v0 account get account1skzn99jx8a2rw80gp9uea47tmcndv7gm28vy9as6akevy32204aqwnfj0ln --host http://127.0.0.1:46127/api --output-format json | jq -c
{"counter":0,"delegation":{"pools":[["5cfbb7f7ec0050499fbd1c79515216c5ad636b97d1944bde8a7db98e7d126803",1]]},"value":1}
```
The result is exactly the same.

Internally, these two `bech32` strings resolve to the same account.

Reply:
Yes! The human readable part ought to be ignored by softwares for it is really for.. humans :sweat_smile:
`account` sounds like a good choice of prefix in this situation :+1:

Reply:
@KtorZ wrote:

> I'd suggest using something that is more representative of an account public key.

:+1: 

I've changed the prefix to "account". See: https://github.com/input-output-hk/cardano-wallet/pull/995/commits/1639467c242a5f7e769cfb47e28dd6fef32d8a8f

Though of course if this isn't suitable, let me know, and we can change it to something else.